LPS-195720 Add aria label for feature flag toggle

SelenaAungst opened the pull request:
https://liferay.atlassian.net/browse/LPS-195720

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-40.liferay.com/job/test-portal-source-format/2885/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/2885/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ece66430e068f9d986100c239e5c49e66f11a079">ece66430e068f9d986100c239e5c49e66f11a079</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/SelenaAungst/liferay-portal/tree/LPS-195720">LPS-195720</a><br/>Branch GIT ID: <a href="https://github.com/SelenaAungst/liferay-portal/commit/8b6d18f76ea5a66e63563d980ea56d59faffba28">8b6d18f76ea5a66e63563d980ea56d59faffba28</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-40.liferay.com/job/test-portal-source-format/2885/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/2885/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/7832/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7832/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-40.liferay.com/job/test-portal-source-format/2885/">test-portal-source-format#2885</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-40.liferay.com/userContent/jobs/test-portal-source-format/builds/2885/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-peds/liferay-portal/pull/261">liferay-peds#261</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2424662016">[master] ci:test:sf - SelenaAungst &gt; liferay-peds - PR#261 - 2023-09-14[15:25:21]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-40.liferay.com/job/publish-testray-report/11101/">publish-testray-report#11101</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 30 out of 30 jobs passed</h3><h3>:x: ci:test:relevant - 57 out of 61 jobs passed in 2 hours 28 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ece66430e068f9d986100c239e5c49e66f11a079">ece66430e068f9d986100c239e5c49e66f11a079</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ece66430e068f9d986100c239e5c49e66f11a079">ece66430e068f9d986100c239e5c49e66f11a079</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2424477040">EE Development Acceptance (master) - 14 - 2023-09-14[09:16:44]</a></p><details><summary><strong>ci:test:stable - 30 out of 30 jobs PASSED</strong></summary><details><summary><strong>30 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 57 out of 61 jobs PASSED</strong></summary><div><h4>4 Failed Jobs:</h4><ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/7832/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>build-iw-jdk8<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/262237/">build-iw-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/275174/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-mysql57-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/693184/">empty-osgi-core-dir-mysql57-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-db2115-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/638559/">functional-smoke-tomcat90-db2115-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-hypersonic20-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/766295/">functional-smoke-tomcat90-hypersonic20-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/285979/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/270925/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-tomcat90-oracle193-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/670480/">functional-smoke-tomcat90-oracle193-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265422/">functional-smoke-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql153-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/650242/">functional-smoke-tomcat90-postgresql153-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-sqlserver2019-jdk8<ul><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/283848/">functional-smoke-tomcat90-sqlserver2019-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/734485/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/270923/"><strike>functional-smoke-wildfly180-mariadb106-jdk8/0/0</strike></a></li></ul></li><li>functional-tomcat90-mysql57-chrome100-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/742368/">functional-tomcat90-mysql57-chrome100-jdk8/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/670479/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/819616/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/742367/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/766297/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/249613/">functional-upgrade-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/257416/"><strike>functional-upgrade-tomcat90-mysql57-jdk8/0/5</strike></a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/262818/">functional-upgrade-tomcat90-mysql57-jdk8/0/6</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/712926/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327067/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/670478/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/2</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/262817/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/3</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/342702/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/4</a></li></ul></li><li>functional-upgrade-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/275171/">functional-upgrade-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-postgresql153-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/766294/">functional-upgrade-tomcat90-postgresql153-jdk8/0/0</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/249611/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-mysql57-jdk8<ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/232641/">lpkg-controller-mysql57-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268658/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268039/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/596127/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/720153/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/283849/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/270924/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/235109/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268040/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/267613/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/266449/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/285978/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/265423/">modules-compile-jdk8/0/11</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/638458/"><strike>modules-integration-mysql57-jdk8/0/0</strike></a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/580533/">modules-integration-mysql57-jdk8/0/1</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/819617/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/734488/">modules-integration-mysql57-jdk8_stable/0/1</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/734486/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/287152/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/249614/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/301647/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/843874/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-mysql57-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/819614/">patching-tool-mysql57-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/638457/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-mysql57-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/580532/">portal-license-smoke-mysql57-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/270926/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/580530/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/268659/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/734487/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/670477/">unit-jdk8_stable/0/0</a></li></ul></li><li>workspaces-compile-jdk8<ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/266448/">workspaces-compile-jdk8/0/0</a></li></ul></li></ul></div><details><summary><strong>57 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7832/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/257416//consoleText">functional-upgrade-tomcat90-mysql57-jdk8/0/5</a><pre>print-jstack-logs:
     [echo] Executing commands: 
     [echo] 
     [echo] jps &gt; jps_output
     [echo] 
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/jps_output
[stopwatch] [test.execution.duration: 16:36.994 sec]
[stopwatch] [run.batch.test.action: 17:22.998 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:373: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-batch.xml:1762: /opt/dev/projects/github/liferay-portal/build-test-batch.xml:1766: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-tomcat.xml:40: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test-tomcat.xml:52: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test.xml:15602: The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/build-test.xml:16831: Timeout: killed the sub-process
      [get] Getting: \$\{dependencies.url\}/archive/test-1-7/test-portal-acceptance-pullrequest-downstream(master)/257416consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20230914165730895.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20230914165730895.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1256851215.properties</pre><ol><li><div><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/257416//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmokeUpgrade_ViewPortalSmokeArchive7310_">LocalFile.PortalSmokeUpgrade#ViewPortalSmokeArchive7310</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-09/test-1-30/test-portal-acceptance-pullrequest(master)/7832/functional-upgrade-tomcat90-mysql57-jdk8/0/5/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive7310/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-09/test-1-30/test-portal-acceptance-pullrequest(master)/7832/functional-upgrade-tomcat90-mysql57-jdk8/0/5/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive7310/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-09/test-1-30/test-portal-acceptance-pullrequest(master)/7832/functional-upgrade-tomcat90-mysql57-jdk8/0/5/LocalFile.PortalSmokeUpgrade_ViewPortalSmokeArchive7310/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-09/test-1-30/test-portal-acceptance-pullrequest(master)/7832/functional-upgrade-tomcat90-mysql57-jdk8/0/5/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2424240338/history">Failed 1 of last 23</a></div></li></ol></div></li><li><div><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/270923//consoleText">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a><ol><li><details><summary><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/270923/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/270923//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li><li><div><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/270923//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_PortalSmoke_Smoke_">LocalFile.PortalSmoke#Smoke</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-09/test-1-30/test-portal-acceptance-pullrequest(master)/7832/functional-smoke-wildfly180-mariadb106-jdk8/0/0/LocalFile.PortalSmoke_Smoke/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-09/test-1-30/test-portal-acceptance-pullrequest(master)/7832/functional-smoke-wildfly180-mariadb106-jdk8/0/0/LocalFile.PortalSmoke_Smoke/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-09/test-1-30/test-portal-acceptance-pullrequest(master)/7832/functional-smoke-wildfly180-mariadb106-jdk8/0/0/LocalFile.PortalSmoke_Smoke/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-09/test-1-30/test-portal-acceptance-pullrequest(master)/7832/functional-smoke-wildfly180-mariadb106-jdk8/0/0/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2424231848/history">Failed 0 of last 23</a></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at ece66430e068f9d986100c239e5c49e66f11a079:</strong></summary><ol><li><div><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/638458//consoleText">modules-integration-mysql57-jdk8/0/0</a><ol><li><details><summary><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/638458/testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest">com.liferay.portal.log.assertor.PortalLogAssertorTest</a></summary><div><ol><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/638458//testReport/com.liferay.portal.log.assertor/PortalLogAssertorTest/testScanXMLLog">testScanXMLLog</a></li></ol></div></details></li></ol></div></li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/7832/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/7832/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-30/jobs/test-portal-acceptance-pullrequest(master)/builds/7832/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest(master)/7832/">test-portal-acceptance-pullrequest(master)#7832</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-30.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7832/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-peds/liferay-portal/pull/261">liferay-peds#261</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2425014215">[master] ci:test:relevant - SelenaAungst &gt; liferay-peds - PR#261 - 2023-09-14[15:40:50]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-30.liferay.com/job/publish-testray-report/18957/">publish-testray-report#18957</a><br/></div></div>

john-co commented:
Reviewed. Test failures are unrelated :heavy_check_mark: 

Ok to forward from QA perspective. Pending dev approval.

marcoscv-work commented:
LGTM!

ethib137 commented on modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagToggle.tsx:
```diff
@@ -61,9 +65,10 @@ const FeatureFlagToggle = ({
 	return (
 		<>
 			<ClayToggle
+				aria-label={featureFlagTitle + featureFlagLabel}
```
We can't join two language keys like this because it will cause them to be read differently for LTR and RTL languages. Basically for LTR it would read `LPS-12345 Disabled` and for RTL it would read `Disabled LPS-12345`.

There are two options to handle this... one is to join them using a language key like `x-x`, then the string can be properly inverted for RTL. The other method is to use another way of adding the extra information, like using `aria-describedby`. Something like [this example](https://github.com/liferay/liferay-portal/commit/3016c187ffcf9365ace51f6651792e60402b5689). I would recommend going for aria-describedby since I think it will be much simpler in this situation.

ethib137 reviewed (changes requested):
Thanks @SelenaAungst ! Just a couple small changes needed. 😄

SelenaAungst commented:
Hi @ethib137,
Thanks for your feedback on this! I added the changes you suggested. I noticed that [another commit](https://github.com/liferay-peds/liferay-portal/pull/261/commits/3207b82cebf8e65942925a10f8016e5d930640a3) was added when I added your changes, is this okay to include or should I resend?

ethib137 commented on modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/toggle_switch.jspf:
```diff
@@ -9,6 +9,7 @@
 String labelOff = LanguageUtil.get(request, "disabled");
 String labelOn = LanguageUtil.get(request, "enabled");
 
+String featureFlagTitle = featureFlagDisplay.getTitle() + " ";
```
You should be able to remove this line now.

ethib137 reviewed (changes requested):
Hey @SelenaAungst , the changes look good. We prefer to not include merge commits, so you can you remove that force push? Thanks :D

ethib137 reviewed (approved):
LGTM. Thanks @SelenaAungst

john-co commented:
cc/ @SelenaAungst please rebase, looks good to `ci:forward`

SelenaAungst commented:
Thanks! @john-co @ethib137 resent here: https://github.com/liferay-peds/liferay-portal/pull/269
